data.json

```diff
@@ -129,10 +129,10 @@
     "name": "Datenanfragen.de e. V.",
     "difficulty": "medium",
     "description": "One merged PR gets at least 3 stickers (limited to 100 sets) and a chance at a limited edition t-shirt.",
-    "reference": "https://www.datarequests.org/blog/hacktoberfest-2019/",
-    "image": "https://www.datarequests.org/blog/hacktoberfest-2019/hacktoberfest-2019.png",
+    "reference": "https://www.datarequests.org/blog/hacktoberfest-2020/",
+    "image": "https://www.datarequests.org/blog/hacktoberfest-2020/hacktoberfest-2020_hu28b1cba020bf43bc24568088e4124bec_241852_881x0_resize_q75_box.jpg",
     "dateAdded": "2019-10-23T00:39:55.000Z",
```

Comment:
Please bump the date so it goes first in the list.

Reply:
```suggestion
    "dateAdded": "2020-10-01T19:00:00.000Z",
```